examples/basic/mining_nar.py

```diff
@@ -0,0 +1,128 @@
+import desbordante
+import pandas
+from tabulate import tabulate
+
+TABLE = 'examples/datasets/dog_breeds.csv'
+CLR = {
+    'bold_default': '\033[1m',
+    'bold_red': '\u001b[1;31m',
+    'bold_green': '\033[1;32m',
+    'bold_yellow': '\033[1;33m',
+    'bold_blue': '\033[1;34m',
+    'gray': '\033[90m',
+    'default': '\033[0m',
+    'blue_bg': '\033[1;46m',
+    'default_bg': '\033[49m',
+    'red_bg': '\033[41m'
+}
+
+def print_table(table, title=None):
+    if title is not None:
+        print(title)
+
+    print(tabulate(table, tablefmt='pipe', stralign='center'))
+
+
+def print_map(map, table):
+    column_names = list(table.columns)
+    for key, value in map.items():
+        print(column_names[key], end='')
+        print(value)
+
+
+def print_10_discovered_nars(algo, table):
+    counter = 1
+    for nar in algo.get_nars()[:10]:
+        print("NAR " + str(counter) + f":{CLR['bold_default']}")
+        counter += 1
+        print_map(nar.ante, pdtable)
+        print("      |\n      |\n      V")
+        print_map(nar.cons, pdtable)
+        print("   support = ", nar.support)
+        print("   confidence = ", nar.confidence)
```

Comment:
```suggestion
        print(f"   {nar.support = }")
        print(f"   {nar.confidence = }")
```

Reply:
This is a debug format, doesn't make sense here. Use this instead.
```suggestion
        print(f"   support = {nar.support}")
        print("   confidence = {nar.confidence}")
```